### ISSUE TITLE
SWMM_MTP4-r2_Import/export plugin and file menus

### ISSUE THREAD

**Issue by barrc** [USEPA/SWMM-EPANET_User_Interface#174]
- Are the ImportExportGIS plugin and the Import/Export submenus accessed via the File menu supposed to have the same functionality?  If so why is there a separate plugin?
- Why are subcatchments exported to a separate json file from the rest of the model data?
- Exporting as geojson, shapefile, or csv doesn't export all the data.
- Importing subcatchments didn't import their names, not all subcatchments were imported, and some data was imported incorrectly.
- Under ImportExportGis -> Import from GIS to create model, the file type menu has options for .shp, .csv, and .gdb, but if you try them you get a message saying 'Create model from GIS data supports geojson data only'
- File -> Import -> Scenario: get popup saying not yet implemented

**Comment by barrc** [comment]
Should also document the format of the geojson required for import (or link to the documentation, if it already exists).

**Comment by PaulDudaRESPEC** [comment]
- Yes, the ImportExportGIS plugin and the Import/Export submenus do have the same functionality.  The functionality was included as a plugin during early development as a demonstration of how a user could develop a plugin within the context of this project.  It's a fair question to ask if it should continue to be included both ways.
- We've now enhanced the code to put the subcatchments in the same geojson as the other layers, I think this is preferred.
- I tested the export to geojson, shapefile, and csv using the SWMM tutorial1.inp and it appears all data is now exported correctly.
- I tested the import from geojson using the above file and it appears all data is imported correctly.
- Import from GIS is only available from geojson, so we removed the .shp, gdb, and .csv options from the file dialog.
- File:Import:Scenario is indeed not implemented as it has been considered lower priority -- a user can do the essentially the same thing by saving the project under another name.
- The geojson format is standard geojson, but indeed it should be documented.  Since that's not a bug per se, I'm going to change the label for this issue to 'enhancement'.
